tests/phpunit/Patcher/ListPatcherTest.php

```diff
@@ -91,9 +92,28 @@ public function patchProvider() {
 
 		$argLists[] = array( $patcher, $base, $diff, $expected );
 
+		$patcher = new ListPatcher();
+		$base = array(
+			$this->newObject( 'foo' ),
+			$this->newObject( 'bar' ),
+		);
+		$diff = new Diff( array(
+			new DiffOpRemove( $this->newObject( 'foo') ),
+			new DiffOpAdd( $this->newObject( 'baz' ) ),
+		) );
+		$expected = array( $this->newObject( 'bar' ), $this->newObject( 'baz' ) );
+
+		$argLists[] = array( $patcher, $base, $diff, $expected );
+
 		return $argLists;
 	}
 
+	private function newObject( $value ) : stdClass {
+		$object = new stdClass();
+		$object->element = $value;
+		return $object;
```

Comment:
`return (object)[ 'element' => $value ]` is a clever 1-liner to do the same. ;-)

Reply:
This way you also avoid the assignment to a field that was not explicitly defined